src/components/HomeEmptyState/index.module.scss

```diff
@@ -0,0 +1,18 @@
+.footer {
+    height: 1.75rem;
+    width: 3rem;
+    margin-bottom: 1.438rem;
+}
+
+.containerHomeEmpty {
```

Comment:
homeEmptyContainer makes more sense to me

Reply:
Fixed!